onboard/catkin_ws/src/motion_planning/scripts/move_tasks.py

```diff
@@ -11,15 +11,16 @@ class MoveToPoseGlobalTask(Task):
     def __init__(self, x, y, z, roll, pitch, yaw, *args, **kwargs):
         super(MoveToPoseGlobalTask, self).__init__(*args, **kwargs)
 
-        self.desired_pose = Pose()
-        self.desired_pose.position.x = x
-        self.desired_pose.position.y = y
-        self.desired_pose.position.z = z
-        self.desired_pose.orientation = Quaternion(*quaternion_from_euler(roll, pitch, yaw))
+    	self.desired_pose = Pose()
+    	self.desired_pose.position.x = x
+    	self.desired_pose.position.y = y
+    	self.desired_pose.position.z = z
```

Comment:
Could this be simplified to `self.desired_pose.position = Point(x=x, y=y, z=z)` or something along those lines?